Google.PowerShell/Container/GkeClusterCmdlets.cs

```diff
@@ -0,0 +1,155 @@
+﻿// Copyright 2015-2017 Google Inc. All Rights Reserved.
+// Licensed under the Apache License Version 2.0.
+
+using Google.PowerShell.Common;
+using Google.Apis.Container.v1;
+using Google.Apis.Container.v1.Data;
+using System.Management.Automation;
+using System.Collections.Generic;
+using System.Net;
+
+namespace Google.PowerShell.Container
+{
+    /// <summary>
+    /// <para type="synopsis">
+    /// Gets Google Container Clusters.
+    /// </para>
+    /// <para type="description">
+    /// Gets Google Container Clusters. If -Project parameter is not specified, the default project will be used.
+    /// If -Zone or -ClusterName is not used, the cmdlet will return every clusters in every zone in the project.
+    /// If -Zone is used without -ClusterName, the cmdlet will return every clusters in the specified zone.
+    /// If -ClusterName is used without -Zone, the cmdlet will return the specified clusters in the default zone
+    /// (set in Cloud SDK Config). If -Clustername is used with -Zone, the cmdlet will return the specified
+    /// clusters in the specified zone.
+    /// </para>
+    /// <example>
+    ///   <code>PS C:\> Get-GkeCluster</code>
+    ///   <para>Lists all container clusters in the default project.</para>
+    /// </example>
+    /// <example>
+    ///   <code>PS C:\> Get-GkeCluster -Zone "us-central1-a" -Project "my-project"</code>
+    ///   <para>Lists all container clusters in zone us-central1-a for the project "my-project".</para>
+    /// </example>
+    /// <example>
+    ///   <code>PS C:\> Get-GkeCluster -ClusterName "my-cluster"</code>
+    ///   <para>Gets the cluster "my-cluster" in the default zone of the default project.</para>
+    /// </example>
+    /// <example>
+    ///   <code>PS C:\> Get-GkeCluster -ClusterName "my-cluster", "my-cluster-2" -Zone "us-central1-a"</code>
+    ///   <para>
+    ///   Gets the cluster "my-cluster", "my-cluster-2" in the zone "us-central1-a" of the default project.
+    ///   </para>
+    /// </example>
+    /// <para type="link" uri="(https://cloud.google.com/compute/docs/reference/latest/machineTypes#resource)">
+    /// [Machine Type resource definition]
```

Comment:
This link looks like a copy-paste error.
Should it be https://cloud.google.com/container-engine/reference/rest/v1/projects.zones.clusters#Cluster?